streamingvisitors/src/vespa/searchvisitor/searchvisitor.cpp

```diff
@@ -240,9 +240,9 @@ void SearchVisitor::init(const Parameters & params)
         LOG(debug, "QFLAG_DUMP_FEATURES: %s", _rankController.getDumpFeatures() ? "true" : "false");
     }
 
-    if (params.lookup("rankproperties", valueRef) && valueRef.size() > 0) {
+    if (params.lookup("rankproperties", valueRef) && ! valueRef.empty()) {
         LOG(spam, "Received rank properties of %zd bytes", valueRef.size());
-        uint32_t len = static_cast<uint32_t>(valueRef.size());
+        uint32_t len = uint32_t(valueRef.size());
```

Comment:
could just construct len directly from valueRef.size() instead of using a temporary

Reply:
Fixed